### PR TITLE
pciutils: add port

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

**Comment by Dennisbonke** [comment]
Pretty sure this is blocked on managarm/mlibc#688 no?

**Comment by no92** [comment]
Ah yes, I forgot.

**Comment by no92** [comment]
Once managarm/managarm#597 is merged this is good to go.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**
